codeguru_profiler_agent/sdk_reporter/sdk_reporter.py

```diff
@@ -10,14 +11,14 @@
 from codeguru_profiler_agent.sdk_reporter.profile_encoder import ProfileEncoder
 
 logger = logging.getLogger(__name__)
-
+HANDLER_ENV_NAME_FOR_CODEGURU = "HANDLER_ENV_NAME_FOR_CODEGURU"
```

Comment:
How about ending the variable in KEY, or ENV_KEY to be clearer?

Reply:
done!